recipes/kitchen/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "kitchen" %}
+{% set version = "1.2.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kitchen-{{ version }}.tar.gz
+  sha256: b84cf582f1bd1556b60ebc7370b9d331eb9247b6b070ce89dfe959cba2c0b03c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - kitchen
+    - kitchen.collections
+    - kitchen
+    - kitchen.versioning
+    - kitchen.i18n
+    - kitchen.iterutils
+    - kitchen.collections
+    - kitchen.text
+    - kitchen.pycompat24
+    - kitchen.pycompat24.base64
+    - kitchen.pycompat24.sets
+    - kitchen.pycompat25
+    - kitchen.pycompat25.collections
+    - kitchen.pycompat27
+    - kitchen.pycompat27.subprocess
+
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://fedorahosted.org/kitchen
```

Comment:
This link doesn't work any more. It looks like the code lives at https://github.com/fedora-infra/kitchen currently, so maybe change to that.

Reply:
Link updated